app/controllers/hyrax/content_blocks_controller.rb

```diff
@@ -3,27 +3,32 @@ class ContentBlocksController < ApplicationController
     load_and_authorize_resource except: :index
     before_action :load_featured_researchers, only: :index
     authorize_resource only: :index
+    layout 'dashboard'
 
-    def index; end
-
-    def create
```

Comment:
tbh, I'm not sure that we ever exposed creation of new content blocks in the UI, so I removed.